### ISSUE TITLE
We should have a central Accessibility.md page for the project

### ISSUE THREAD

**Issue by mgifford** [usds/ditap-curriculum-update#257]
**Definition of success**
- just needs to be something on https://github.com/usds/ditap-curriculum-update/i

**Other information**
- Needs to align with 508.

**Additional context**
- Need to address binary files (pdf, ppt, doc, etc.)
- Highlighting the contributors
- Plain language targets
- Automated testing (if possible)
- Feedback loops for users

**Comment by mgifford** [comment]
Created #258
